src/main/java/frc/robot/subsystems/PivotSparkMax.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import org.littletonrobotics.junction.Logger;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMax.ControlType;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.revrobotics.SparkMaxAbsoluteEncoder.Type;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.Constants.PivotConstants;
+
+public class PivotSparkMax implements PivotIO {
+  private final CANSparkMax leftMotor;
+  private final CANSparkMax rightMotor;
+  private final AbsoluteEncoder leftEncoder;
+  private final AbsoluteEncoder rightEncoder;
+  private final SparkMaxPIDController leftController;
+  private final SparkMaxPIDController rightController;
+  private final PivotIOInputsAutoLogged inputs = new PivotIOInputsAutoLogged();
+  /** Creates a new PivotReal. */
+  public PivotSparkMax(int leftMotorID, int rightMotorID) {
+    this.leftMotor = new CANSparkMax(leftMotorID, MotorType.kBrushless);
```

Comment:
Try to control the motors together -- you can set one as a follower motor. Keep inversions in mind